front/pages/api/v1/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -285,16 +285,19 @@ async function handler(
         "App run creation"
       );
 
-      const runRes = await coreAPI.createRunStream({
-        projectId: app.dustAPIProjectId,
-        runAsWorkspaceId: keyWorkspaceId,
-        runType: "deploy",
-        specificationHash: specificationHash,
-        config: { blocks: config },
-        inputs,
-        credentials,
-        secrets,
-      });
+      const runRes = await coreAPI.createRunStream(
+        auth.getNonNullableWorkspace(),
```

Comment:
Don't we already have `owner` from above ?

Reply:
Ah yeah good catch. Let me fix.